src/main/java/org/folio/rest/impl/BLUsersAPI.java

```diff
@@ -802,6 +807,7 @@ public void postBlUsersLogin(boolean expandPerms, List<String> include, String u
     }
 
     if (entity == null || entity.getUsername() == null || entity.getPassword() == null) {
+      client.closeClient();
```

Comment:
The next catch clause (line 828) also needs a close.

Reply:
Damn. A very bad style